R/matchDim.R

```diff
@@ -1,32 +1,51 @@
-#' match object dimensions of a magclass object to the dimensions of a reference object
+#' Match dimensions of a magpie object to those of a reference object
 #'
-#' A helper that restricts and expands a magclass object x to the size of a magclass object ref.
-#' Dimension names not present in x are added and set to the value provided by 'fill'.
-#' Dimension names not present in ref are cropped.
+#' A helper that restricts and expands a magclass object `x` to the size of a
```

Comment:
Shouldn't this be renamed to "magpie" here, too?

Reply:
Sure does/